src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -108,6 +108,12 @@ object ExtractionPipeline {
       linkingEntities = replicates
     )
 
+    val fc_experiments = extractLinkedEntities(
```

Comment:
```suggestion
    val fcExperiments = extractLinkedEntities(
```
Scala style

Reply:
Switched this to "fcExperiments" throughout the script